displayCavity.py

```diff
@@ -31,19 +33,20 @@ def __init__(self, cavityNum, rackObject):
                                                   RACK=self.rack.rackName,
                                                   CAVITY=self.number)
 
-            self.faults.append(Fault(tlc=csvFault["Three Letter Code"],
+            tlc = csvFault["Three Letter Code"]
+            self.faults[tlc] = Fault(tlc=tlc,
```

Comment:
What was the advantage of line 36 here? Was this a necessary step to help turn faults into a dictionary as opposed to a list like it was before?

Reply:
I can't see the lines below it, but I think I use tlc again? I typically pull things into variables if I use them more than once haha